src/styles/_catalog.scss

```diff
@@ -1,22 +1,37 @@
+$card-width: 200px;
+$card-gap: 48px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $card-gap;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+
+  @media (max-width: 1023px) {
```

Comment:
you were told to use 1024px, 768px, 488px — so let's apply these numbers (neither less, nor greater)